### DISCUSSION TITLE
Github Contributions Table Error

### DISCUSSION THREAD

**Discussion by devisasari** [community/community#36037] [Projects and Issues] [answered]
I have been using Github for a long time. In the contribution table, I noticed that the commits I made these days were merged. I told a friend of mine to check his own account to see if he also has this issue and we realized that he had the same problem. My friend and I both got the "server error 500" on October 6th. Both my friend's and my February commits are wrong. The statistics I see in the table when I login to my account ([github.com/devisasari](http://github.com/devisasari)) and the statistics I see in the table when I am not logged in are different. Please check the screenshots in ss-1 and ss-2 to see this issue. (The commit of February 11 is on February 12).
 
We've included screenshots showing these issues. If you look at these accounts with and without login, you will see the differences. Also, if you check your official twitter account, [twitter.com/githubstatus](http://twitter.com/githubstatus), you can see that there were problems on the dates we mentioned earlier. We have added the proofs of this issue as ss-7, ss-8, and ss-9.

![ss-2](https://user-images.githubusercontent.com/99514353/195626425-db3a189a-e7c7-4dd1-bf19-1b36d82ab4f5.png)
<img width="444" alt="ss-7" src="https://user-images.githubusercontent.com/99514353/195626438-9fbc576c-2088-4371-a1f8-f126befe0a2c.png">
<img width="455" alt="ss-8" src="https://user-images.githubusercontent.com/99514353/195626442-5eb01fca-1fbe-4e7a-94f1-69066978e31a.png">
<img width="453" alt="ss-9" src="https://user-images.githubusercontent.com/99514353/195626451-aea551c9-2274-4a42-bec8-65ce005f1509.png">
![ss-1](https://user-images.githubusercontent.com/99514353/195626455-522c7431-af86-4111-b437-9718895eb0a7.png)

**Comment by kaplanark** [comment, marked as answer]
If you wish to restore those deleted repositories, which will restore any associated contributions, you should be able to restore that repository yourself by following these steps:
[Restoring a deleted repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository)
If any of those deleted repositories were forks then you will not be able to restore them yourself. In that case please let us know Github and they can restore them for you.

**Comment by saikrishnadas** [comment]
Even for me, the contribution table doesn't track the commits even though a lot of commits are made to my repo. It was working a few months back.
